cypress/e2e/views/menu.view.ts

```diff
@@ -13,5 +13,5 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-export const navMenu = "a.pf-c-nav__link";
+export const navMenu = 'a[class*="nav__link"]';
```

Comment:
This change looks to be the result of the apps upgrade to PF5. Any changes around the selectors here should be considered permanent. Thanks for your patience with us as we take on this major upgrade with breaking changes. 🙏

Reply:
Thanks @ibolton336 !